src/utils/test/util.spec.ts

```diff
@@ -1,6 +1,49 @@
+import { mockConfig } from '@stencil/core/testing';
 import * as util from '../util';
 
 describe('util', () => {
+  describe('generatePreamble', () => {
+    it('generates a comment with a single line preamble', () => {
+      const testConfig = mockConfig();
+      testConfig.preamble = 'I am Stencil';
+
+      const result = util.generatePreamble(testConfig);
+
+      expect(result).toBe(`/*!
+ * I am Stencil
+ */`);
+    });
+
+    it('generates a comment with a multi-line preamble', () => {
+      const testConfig = mockConfig();
+      testConfig.preamble = 'I am Stencil\nHear me roar';
```

Comment:
I omitted the test case where we have something with a trailing newline:
```tsx
      testConfig.preamble = 'I am Stencil\nHear me roar\n';
```

I didn't think it added _too_ much value - LMK if you think differently though